### PR TITLE
detray: add v0.70.0, v0.71.0, v0.72.1

### PR DESCRIPTION
**Pull request by stephenswat**
This commit adds four new versions of the detray package.

### CODE REVIEW

File: var/spack/repos/builtin/packages/detray/package.py [hunk 1]
```diff
@@ -20,7 +20,11 @@ class Detray(CMakePackage):
 
     license("MPL-2.0", checked_by="stephenswat")
 
+    version("0.72.1", sha256="6cc8d34bc0d801338e9ab142c4a9884d19d9c02555dbb56972fab86b98d0f75b")
+    version("0.71.0", sha256="2be2b3dac6f77aa8cea033eba841378dc3703ff93c99e4d05ea03df685e6d508")
+    version("0.70.0", sha256="14fa1d478d44d5d987caea6f4b365bce870aa8e140c21b802c527afa3a5db869")
     version("0.69.1", sha256="7100ec86a47458a35f5943cd6c7da07c68b8c1c2f62d36d13b8bb50568d0abe5")
+    version("0.69.0", sha256="d7de190c0c5af4567dc5cd30913be649b1652ebe8961e514484b99aaabd3051a")
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/detray/package.py]: I noticed you added 0.72.1 in favor of 0.72.0 (in line with the usual practice in spack of adding the latest patch level). Did you intend to add 0.69.0 when 0.69.1 is already there?

**Comment by wdconinc** [comment]
Verified all hashes.

File: var/spack/repos/builtin/packages/detray/package.py (hunk 1 above)
Comment (stephenswat) [reply var/spack/repos/builtin/packages/detray/package.py]: Indeed this isn't really necessary. I've removed it!

**Review by wdconinc — APPROVED** [review]

**Comment by wdconinc** [comment]
@spackbot run pipeline

**Pull request merged**